plasma_framework/test/helpers/constants.js

```diff
@@ -14,6 +14,9 @@ const OUTPUT_TYPE = {
 const EMPTY_BYTES = '0x';
 const EMPTY_BYTES_32 = '0x0000000000000000000000000000000000000000000000000000000000000000';
 
+const ETH_VAULT_ID = 1;
```

Comment:
I would probably do the following instead:
```
VAULT_ID = {
    ETH: 1,
    ERC20: 2,
}
```

It is more consistent with how we treat `TX_TYPE` and `OUTPUT_TYPE`

Reply:
done